src/sentry/incidents/models.py

```diff
@@ -578,10 +579,7 @@ class Meta:
     @property
     def target(self):
         if self.target_type == self.TargetType.USER.value:
-            try:
-                return User.objects.get(id=int(self.target_identifier))
-            except User.DoesNotExist:
-                pass
+            return user_service.get_user(int(self.target_identifier))
```

Comment:
Does this still swallow errors on missing users?

Reply:
Yes

Reply:
In, this case it happens to be the same behavior in both cases because there is no other pass through logic (because the `pass` here is really a `return None` in practice.